Description: Do not make the *.pyc and *.pyo files and use
Debian's packaging layout when installing the package.
Author: Ana Guerrero <ana-externe.guerrero@edf.fr>
Last-Update: 2015-01-22

--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,8 @@ all: $(MANPAGE)
 	$(PYTHON) setup.py build

 install: all
-	$(PYTHON) setup.py install -O1 --skip-build --root $(DESTDIR)
+	$(PYTHON) setup.py install --skip-build --root $(DESTDIR) --no-compile --install-layout=deb
+
 	# config files
 	install -d $(DESTDIR)/$(SYSCONFIGDIR)/$(NAME)/conf/samples
 	install -p -m 0644 conf/milkcheck.conf $(DESTDIR)/$(SYSCONFIGDIR)/$(NAME)
